### ISSUE TITLE
Jackett can't do multiple searches at the same time

### ISSUE THREAD

**Issue by thaynos** [qbittorrent/search-plugins#210]
the jackett repo redirected me here 
https://github.com/Jackett/Jackett/issues/13551

**Comment by ngosang** [comment]
qBittorrent plugin for Jackett is using the "aggregate indexer" to make the requests. That endpoint takes a lot of time (if you have many indexers configured) and it can cause problems if you run several queries at the same time.
https://github.com/Jackett/Jackett#aggregate-indexers

It will be fixed if we complete this feature someday => #121

**Comment by galeksandrp** [comment]
You can just split your `jackett.py` search engine to multiple search engines.

1. Open folder where search engines are stored (`%localappdata%\qBittorrent\nova3\engines\` on `Windows`)
2. For each `tracker` configured in `jackett` you need to guess `indexer name`. 
```
Point you mouse to `button` `Copy RSS Feed` in `jackett`. Browser will show you link.
http://127.0.0.1:9117/api/v2.0/indexers/<indexer name>/results/torznab/api?apikey=...
Part between `indexers/` and `/results` will be your `indexer name`.
```
4. Copy file `jackett.py` to `jackett<indexer name>.py`
5. Open file `jackett<indexer name>.py`
6. Replace line
```py
class jackett(object):
```
with
```py
class jackett<indexer name>(object):
```
7. Replace line
```py
name = 'Jackett'
```
with
```py
name = 'Jackett<indexer name>'
```
9. Replace line
```py
jacket_url = self.url + "/api/v2.0/indexers/all/results/torznab/api?%s" % params
```
with
```py
jacket_url = self.url + "/api/v2.0/indexers/<indexer name>/results/torznab/api?%s" % params
```
10. Replace line
```py
jackett_se = jackett()
```
with
```py
jackett_se = jackett<indexer name>()
```
11. Repeat for every `tracker` in `jackett`
12. Restart `qbittorrent`
13. Go to `search providers` settings, you should see truckload of tracker specific `jackett` tracker providers.
14. Now you can `disable` `jackett` provider, since you have truckload of tracker specific `jackett` providers.
